### DISCUSSION TITLE
MAC Command DutyCycleReq

### DISCUSSION THREAD

**Discussion by ghalencar** [Lora-net/LoRaMac-node#1327] [Q&A]
Hello,

sorry for my English.

On some devices I am getting the following MAC command "DutyCycleReq" 0x04 from NS:

  "mac_commands": [
    {
        "DutyCycleReq": {
         "MaxDCycle": 0
    }

The command I understand what it's for. But I don't understand why I get it. Is it likely that I'm sending something non-standard, along with my payload?

**Comment by mluis1** [comment]
It looks like that your network server provider is restricting how often your node can transmit.

You should contact your network server provider in order to know why he is restricting your node transmissions.
